packages/docs/components/atoms/LargeText/index.md

```diff
@@ -1,13 +1,24 @@
-# LargeText (todo) <Badges :texts="badges" />
+# LargeText <Badges :texts="badges" />
 
 <script setup>
   import pkg from '@studiometa/ui/atoms/LargeText/package.json';
   const badges = [`v${pkg.version}`, 'JS', 'Twig'];
 </script>
 
+Use the large text component to easily add very nice looking title animation.
```

Comment:
**improvement** Could maybe have a clearer explanation. Use the LargeText component to implement a horizontal scrolling text on scroll. 
**unrelated**  This component have a name which isnt clear and descriptiive

Reply:
yeah good name !

Reply:
I suggest that we keep this one as is to not break projects using it. We can work on a new `Marquee` component with a more robust API.